tests/test_connect.py

```diff
@@ -98,6 +98,20 @@ def test_tcp_ssl_tls12_custom_ciphers(tcp_address, ssl_ciphers):
     _assert_connect(conn, tcp_address, certfile=certfile, keyfile=keyfile)
 
 
+'''
+Addresses bug CAE-333 which uncovered that the init method of the base
+class did override the initialization of the socket_timeout parameter.
+'''
+def test_unix_socket_with_timeout():
```

Comment:
Two blank-lines required between function definition and doc-blocks, as well as after the test